### PR TITLE
bugfix/394

### LINKED ISSUE #394: PTLs are not correct length
Should be 1 minute in length, but they are something else.

### PR DESCRIPTION
**Pull request by erikquinn**
Resolves #394.

Move PTL length to canvasConstants and set to 1 minute.

### CODE REVIEW

File: src/assets/scripts/client/constants/canvasConstants.js [hunk 1]
```diff
@@ -14,3 +14,21 @@ export const DEFAULT_CANVAS_SIZE = {
...
+/**
+ * Various options for drawing aircraft on canvas
+ *
+ * @enum AIRCRAFT_DRAW_OPTIONS
+ * @type {Object}
+ * @final
+ */
+export const AIRCRAFT_DRAW_OPTIONS = {
+    /**
+     * Length of PTL lines, in minutes. Standard is 1.
```
Comment (n8rzz) [inline src/assets/scripts/client/constants/canvasConstants.js]: what does `PTL` stand for?  that would be a good thing to put in here.

Comment (erikquinn) [inline src/assets/scripts/client/constants/canvasConstants.js]: Sorry, second nature. "Projected Track Lines" is the proper name; it's a setting within the DCB menu of a STARS scope (used in modern FAA TRACONs). I'll amend it to that.

Comment (n8rzz) [inline src/assets/scripts/client/constants/canvasConstants.js]: whats a `DCB`?

Comment (erikquinn) [inline src/assets/scripts/client/constants/canvasConstants.js]: honestly I have no idea actually. it's the name of a big green menu bar across the top that you use to change settings of the scope.

Comment (n8rzz) [inline src/assets/scripts/client/constants/canvasConstants.js]: right on, that last one was half-troll.  I couldn't resist.

Comment (erikquinn) [inline src/assets/scripts/client/constants/canvasConstants.js]: Wha, you think I don't know that? I almost ended my last response with `Oh, and before you ask... FAA = Federal Aviation Administration && STARS = Standard Terminal Automation Replacement System`. _shots fired!_ 💥

Comment (n8rzz) [inline src/assets/scripts/client/constants/canvasConstants.js]: harry-shrugs

**Pull request merged**